contract/src/platform-goals/start-governed-contract.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+import { E } from '@endo/far';
+
+import { allValues } from '../objectTools.js';
+
+/**
+ * @template SF @typedef {import('@agoric/zoe/src/zoeService/utils').StartResult<SF>} StartResult<SF>
+ */
+
+/**
+ * @typedef {StartResult<
+ *   typeof import('@agoric/governance/src/committee.js').prepare
+ * >} CommitteeStart
+ */
+
+// import { CONTRACT_ELECTORATE, ParamTypes } from '@agoric/governance';
+const CONTRACT_ELECTORATE = 'Electorate';
+export const ParamTypes = /** @type {const} */ ({
+  INVITATION: 'invitation',
```

Comment:
IOU a test to check that these constants are in sync with the impoets

Reply:
many / most are tested now

Reply:
The `rollup.config.mjs` that I'm using for core eval scripts so far doesn't grok inter-package module specifiers.
There's a rollup plugin to do that, but I don't (yet) want to give the impression that this is a general module linking approach; for that, folks should use `bundleSource` via `agoric run`.

I did write up docs for this approach:

 - https://github.com/Agoric/documentation/pull/985